lib/util/utils.js

```diff
@@ -314,7 +315,7 @@ exports.mergeObjects = function mergeObjects(source, target) {
       }
       target[key] = exports.mergeArrays(source[key], target[key]);
     } else {
-      target[key] = source[key];
+      target[key] = lodash.cloneDeep(source[key]);
```

Comment:
clone as we do not want the references around as we may modify them independent of each other later.